src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1614,6 +1614,14 @@
 helps['aks nodepool delete'] = """
 type: command
 short-summary: Delete the agent pool in the managed Kubernetes cluster.
+parameters:
+    - name: --ignore-pdb
```

Comment:
> -  FAIL - HIGH severity: unrecognized_help_parameter_rule
>     Help-Entry: `aks nodepool delete` - The following parameter help names are invalid: --ignore-pdb

Try the following, one of them would be correct
```suggestion
    - name: --ignore-pdb --ignore-pod-disruption-budget
```

```suggestion
    - name: --ignore-pod-disruption-budget --ignore-pdb
```

Reply:
Added.